### ISSUE TITLE
[Bug]: New version doesn't work for me

### ISSUE THREAD

**Issue by Sergiobop** [add2cal/add-to-calendar-button#516]
### What happened?
When i try to use the button as i used to in the version 2.4.3, the button doesn't show and in my console i can see this error:
```
TypeError: Cannot create property 'proKey' on string ''
```

### System & Environment

add-to-calendar-button 2.5

**Comment by jekuer** [comment]
Hi @Sergiobop , how are you using the script?

**Comment by arienzIT** [comment]
I'm having the same issue, installing version `2.5.0`on a Nuxt 3 project, the same error is shown and the button is not rendered.
I'm using the web component version, following [official docs](https://add-to-calendar-button.com/use-with-vue).
Here's the code block for completness
```vue
<script setup lang="ts">
import "add-to-calendar-button";
</script>

<template>
  <div class="appointment-toast">
    <add-to-calendar-button
      name="Title"
      options="'Apple','Google'"
      location="World Wide Web"
      start-date="2023-12-19"
      start-time="10:15"
      end-time="23:30"
      time-zone="America/Los_Angeles"
    ></add-to-calendar-button>
  </div>
</template>

```

**Comment by jekuer** [comment]
Should be fixed with v2.5.1.
Problem with your code - you may not use hyphens! If TS complains about camel case, you can also write everything in lower case letters, but using hyphens will fail.
